### DISCUSSION TITLE
404: NOT_FOUND in vercel

### DISCUSSION THREAD

**Discussion by devEGBH** [vercel/community#5948] [Help]
### Summary

![image](https://github.com/vercel/community/assets/122989705/4d88fddd-cfe6-4b3a-8311-8a7bdc3e7686)

site url: https://metashop-peach.vercel.app/

can we find this error why this error is coming?

### Steps to Reproduce

site url: https://metashop-peach.vercel.app/

**Comment by swarnava** [comment]
Hi,

Are you able to run `pnpm run build && pnpm start` locally?

**Reply by devEGBH** [reply]
yes, in local,  it is working but in vercel is not working

**Reply by devEGBH** [reply]
I am using this theme: MetaShop- Headless WooCommerce React Theme (https://jstemplate.net/item/woocommerce-react-theme)

for development - pnpm -F site dev
for build - pnpm -F site build
